### ISSUE TITLE
Fix accessibility issues with the document preview modal

### ISSUE THREAD

**Issue by chasenlehara** [CCALI/A2JViewer#180]
This includes but isn’t limited to:

- [x] When the bubble is first shown, the “open document preview” button should not have focus
- [ ] ~~When the preview modal is opened, an element in the modal should have focus~~
- [x] The close buttons in modal headers should have more contrast
- [x] Focus should go back to the bubble after closing the modal
